docs/concepts/ipfs-implementations.md

```diff
@@ -9,14 +9,20 @@ IPFS is an open-source project that encourages the development of multiple imple
 
 You can learn more about the principles that define what an IPFS implementation is [here](./implementations.md).
 
-::: callout TLDR
-Looking for an easy, user-friendly way to get started with IPFS?
-Try any of the options listed below:
 
-- [IPFS Desktop](../install/ipfs-desktop.md), an user-friendly GUI application that bundles an IPFS Kubo node, file manager, peer manager, and content explorer.
-- [IPFS Companion](../install/ipfs-companion.md), a browser extension that compliments IPFS Desktop, loads compatible websites and file paths from a local IPFS Kubo node.
-- [IPFS Kubo](../install/command-line.md), standalone demon service with command-line and HTTP RPC interface for power users who don't need GUI.
-  :::
+::: callout Mainnet TLDR
```

Comment:
I disagree with including Mainnet here and in line 15. Many of these tools are used for mainnet and also private networks, especially in the Software Development section.

Reply:
Removed mentions of "mainnet" in 06c578416fcabddd8a7bc2db24c16c799dc9a1cc